src/domain/orders/details/create-fulfillment/item-table.tsx

```diff
@@ -1,160 +1,154 @@
-                    </div>
-                  </Table.Cell>
-                  <Table.Cell>
-                    <div className="min-w-[240px] flex py-2">
-                      <div className="w-[30px] h-[40px] ">
-                        <img
-                          className="h-full w-full object-cover rounded"
-                          src={item.thumbnail}
-                        />
-                      </div>
-                      <div className="inter-small-regular text-grey-50 flex flex-col ml-4">
-                        <span>
-                          <span className="text-grey-90">{item.title}</span>
-                        </span>
-                        <span>{item?.variant?.title || ""}</span>
-                      </div>
-                    </div>
-                  </Table.Cell>
-                  <Table.Cell className="text-right w-32 pr-8">
-                    {toFulfill.includes(item.id) ? (
-                      <div className="flex w-full text-right justify-end text-grey-50 ">
-                        <span
-                          onClick={() => handleQuantity(-1, item)}
-                          className={clsx(
-                            "w-5 h-5 flex text-grey-50 items-center justify-center rounded cursor-pointer hover:bg-grey-20 mr-2",
-                            {
-                              ["pointer-events-none text-grey-30"]:
-                                quantities[item.id] === 1,
-                            }
-                          )}
-                        >
-                          <MinusIcon size={16} />
-                        </span>
-                        <span>{quantities[item.id] || ""}</span>
-                        <span
-                          onClick={() => handleQuantity(1, item)}
-                          className={clsx(
-                            "w-5 h-5 flex text-grey-50 items-center justify-center rounded cursor-pointer hover:bg-grey-20 ml-2",
-                            {
-                              ["pointer-events-none text-grey-30"]:
-                                item.quantity - item.fulfilled_quantity ===
-                                quantities[item.id],
-                            }
-                          )}
-                        >
-                          <PlusIcon size={16} />
-                        </span>
-                      </div>
-                    ) : (
-                      <span className="text-grey-40">
-                        {getFulfillableQuantity(item)}
-                      </span>
-                    )}
-                  </Table.Cell>
-                </Table.Row>
-              </>
-            )
-          })}
-      </Table.Body>
-    </Table>
+  React.useEffect(() => {
+    setErrors((errors) => {
+      const newErrors = { ...errors }
+      if (validQuantity) {
+        delete newErrors[item.id]
+        return newErrors
+      } else {
+        newErrors[item.id] = "Quantity is not valid"
```

Comment:
suggestion: line 148 we say `Invalid quantity` and here a different message, should we armosized them?